pkg/types/conf_test.go

```diff
@@ -928,7 +928,13 @@ var _ = Describe("config operations", func() {
 
 		netconf, err := LoadDelegateNetConf([]byte(conf), ns, "", "")
 		Expect(err).NotTo(HaveOccurred())
+
+		err = CheckGatewayConfig([]*DelegateNetConf{netconf})
+		Expect(err).NotTo(HaveOccurred())
+
 		Expect(netconf.GatewayRequest).To(BeNil())
+		Expect(netconf.IsFilterV4Gateway).To(BeEquivalentTo(true))
+		Expect(netconf.IsFilterV6Gateway).To(BeEquivalentTo(true))
```

Comment:
IMO you should use the `BeTrue()` matcher instead. More idiomatic.

The same comment applies below to all other tests.

Reply:
thnx, good point